PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.core/src/com/microsoft/azuretools/core/ui/startup/WACPStartUp.java

```diff
@@ -147,6 +150,10 @@ private void initialize() {
 	        }
 			EventUtil.logEvent(EventType.info, SYSTEM, PLUGIN_LOAD, null, null);
 	        AppInsightsClient.createByType(AppInsightsClient.EventType.Plugin, "", AppInsightsConstants.Load, null, true);
+	        
+	        final String machineId = TelemetryUtils.getMachieId(dataFile, Messages.prefVal, Messages.instID);
+	        final CustomerSurveyHelper helper  = new CustomerSurveyHelper(machineId, curVersion);	        
```

Comment:
We could use `AppInsightsClient.getInstallationId()` to get the machine Id and no need pass it to `CustomerSurveyHelper`

We could also move `Activator.getDefault().getBundle().getVersion().toString();` to Utils and call it in `CustomerSurveyHelper `

Reply:
Done